corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js

```diff
@@ -35,6 +35,12 @@ $(function () {
             // where State.data.tab won't be available
             link = $('a[data-toggle="tab"][href^="' + window.location.pathname + '"]');
             if (link.length !== 0) {
+                if (link.length > 1) {
```

Comment:
Unfortunately, it's hard to tell from the data what is used more since
"settings" is currently the default.

My thoughts are: we're not happy with where "Deploy" is in general. Making
it the default would be a strange experience in the early stages of
building your app (the first, second, and third time you visit the app you
almost certainly want to be in the app builder rather than the release
manager).

I'm pro a totally different kind of nav for "Deploy" (but don't have good
ideas as to what)

On Thu, Feb 25, 2016 at 7:39 AM, Cory Zue <notifications@github.com> wrote:

> In
> corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js
> <https://github.com/dimagi/commcare-hq/pull/10486#discussion_r54095062>:
>
> > @@ -35,6 +35,12 @@ $(function () {
> >              // where State.data.tab won't be available
> >              link = $('a[data-toggle="tab"][href^="' + window.location.pathname + '"]');
> >              if (link.length !== 0) {
> > +                if (link.length > 1) {
>
> could benefit from a comment explaining this (i don't think i totally
> understand it)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10486/files#r54095062>.
>